.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
   - DJANGO_VERSION=1.4.15
   - DJANGO_VERSION=1.6.7
   - DJANGO_VERSION=1.7.0
+  - DJANGO_VERSION=1.8.3
```

Comment:
Would you mind actually matching this up with the officially supported Django versions? Currently 1.4.x, 1.7.x, and 1.8.x - feel free to bump the 1.4.x / 1.7.x to the latest release as well.